### ISSUE TITLE
Support Android Gradle Plugin 8.0

### ISSUE THREAD

**Issue by asaarnak** [flutter-stripe/flutter_stripe#1481]
Gradle propery `android.nonTransitiveRClass` default value was changed to `true`
and [AGP 8.0 generates R classes for resources defined in the current module only.](https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#default-changes:~:text=AGP%208.0%20generates%20R%20classes%20for%20resources%20defined%20in%20the%20current%20module%20only.)

and we get compiler error at [CardFormView.kt:33:71](https://github.com/flutter-stripe/flutter_stripe/blob/c0ec7d7083aed94d79c9ff8693cbaff1134a437d/packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/CardFormView.kt#L33C107-L33C107):
```
e: file:///home/user/.pub-cache/hosted/pub.dev/stripe_android-9.5.0+1/android/src/main/kotlin/com/reactnativestripesdk/CardFormView.kt:33:71 Unresolved reference: style
```

**Comment by asaarnak** [comment]
Found that this is a duplicate issue.
Already fixed by @remonh87 here: https://github.com/flutter-stripe/flutter_stripe/pull/1479/files#diff-3c5f88b974a7b077f26cc3fb96e7dbd365e57592e832917db00641045b9b1273R33

**Comment by asaarnak** [comment]
@remonh87 
Maybe this issue is already addressed here:
https://github.com/flutter-stripe/flutter_stripe/pull/1479

**Comment by paulpivot94** [comment]
Hi, i also have the same issue
e: /home/.pub-cache/hosted/pub.dev/stripe_android-9.5.0+1/android/src/main/kotlin/com/reactnativestripesdk/CardFormView.kt: (33, 71): Unresolved reference: style

please, can somebody help ??

**Comment by remonh87** [comment]
Fixed in Flutter Stripe 9.6.0
